raiden/tests/utils/patch_connection_pool.py

```diff
@@ -0,0 +1,16 @@
+def patch_http_connection_pool(**constructor_kwargs):
```

Comment:
can't we just instantiate a proper poll instead of patching things up?

Reply:
unfortunately we can't - this patches default pool size for libraries like matrix client or web3

Reply:
ahhh darn, yeah, web3's http provider doesn't allow the session to be configured